gammapy/background/energy_offset_array.py

```diff
@@ -0,0 +1,106 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.coordinates import Angle
+from astropy.units import Quantity
+
+__all__ = [
+    'EnergyOffsetArray',
+]
+
+
+class EnergyOffsetArray(object):
+    """
+    Energy offset dependent array.
+
+    Parameters
+    ----------
+    energy : `~astropy.units.Quantity`
+         energy bin vector
+    offset : `~astropy.coordinates.Angle`
+        offset bin vector
+    data : `~numpy.ndarray`
+        data array (2D)
+
+    """
+
+    def __init__(self, energy, offset, data=None):
+        self.energy = Quantity(energy, 'TeV')
+        self.offset = Angle(offset, 'deg')
+        if data is None:
+            self.data = np.zeros((len(energy) - 1, len(offset) - 1))
+        else:
+            self.data = data
+
+    def fill_events(self, event_list):
+        """Fill events histogram.
+
+        This add the counts to the existing value array.
+
+        Parameters
+        -------------
+        events : list of `~gammapy.data.EventList`
+           Python list of event list objects.
+            
+        
+        """
+        # loop over the Lost of object EventList
+        for (i, data_set) in enumerate(event_list):
```

Comment:
You don't need the index `i` here, so this can be simplified to:
```python
for event_list in event_lists:
    counts = self._fill_one_event_list(event_list)
    self.data += counts
```